octoprint_v8theme/static/js/v8theme.js

```diff
@@ -9,6 +9,8 @@ $(function() {
             $("#control_link").addClass("active");
             $("#control").prepend($("#temp").contents());
             $("#control").addClass("active");
+            $("#temperature-graph").closest(".row").removeAttr("style");
+            $("#temperature-graph").closest(".row").attr("class", "row-fluid");
```

Comment:
What you have is probably fine, but when changing classes, `addClass()` and `removeClass()` are usually what you want.  They treat the classes of an element as a set data structure, rather than a string.